### ISSUE TITLE
How to make Mypal68 run smoothly ?

### ISSUE THREAD

**Issue by andika207** [Feodor2/Mypal68#298]
how can I speed up the mouse scrolling and make it run smoother on newspaper' large websites ?

I want it to work as smooth as the 360 chrome browser

I did a search on google but none of the strings for Firefox seems to take effect with Mypal68

**Comment by enigma131** [comment]
What are your os, updates and motherboard?
FYI, I had to patch hal.dll to have smooth scrooling with my H310 motherboard under Xp 64.
Had you made wintimertester.exe test and if yes have you 1.0 as test result ?

**Comment by K4sum1** [comment]
> What are your os, updates and motherboard? FYI, I had to patch hal.dll to have smooth scrooling with my H310 motherboard under Xp 64. Had you made wintimertester.exe test and if yes have you 1.0 as test result ?

hal.dll patch? What does it do? What is the patch?

**Comment by andika207** [comment]
> What are your os, updates and motherboard? 

what has the hardware to do with Mypal68 scrolling experience ?

360chrome is smooth out of the box so I wanted to know how to match this on Mypal68

**Comment by enigma131** [comment]
Make what you will ... You diden't gave me the response about wintimertester... so I can't say more.

**Comment by andika207** [comment]
> You diden't gave me the response about wintimertester... so I can't say more.

I could not care less about your wintishit...
if you don't know the way to tinker around with Mypal68 settings just refrain from posting on my topics.

you better off write in your native language so I can use edge or google to translate it for me (if necessary).
I have no clue what you are talking about dude

**Comment by Feodor2** [comment]
You did not give a single link to check - nothing to do

**Comment by andika207** [comment]
any link of your choice. for example [dailymail.co.uk](url) 
the scroll bar is not smooth and thumbnails take longer to load than they do with chrome
I know there was a parameter on Firefox 52 to smoothing the scroll bar but can't remember which one

**Comment by enigma131** [comment]
dailymail works perfectly here

**Comment by IP-CAM** [comment]
> dailymail works perfectly here

here too !

**Comment by andika207** [comment]
> here too !

do you use any chrome based browser ?

**Comment by enigma131** [comment]
Works perfect, 360 browser or Gecko based browsers, smooth scrolling with both types.
We can't help you more without your hardware / Os.

**Comment by andika207** [comment]
> Works perfect, 360 browser or Gecko based browsers, smooth scrolling with both types. 

Ok I found a fix here https://askubuntu.com/questions/1447009/control-scrolling-speed-in-webpages-in-firefox

the Mypal68' default value is 100 so I set it to 500
the thumbnails now load faster and the scroll bar feels way smoother

**Comment by andika207** [comment]
>  I had to patch hal.dll to have smooth scrooling with my H310 motherboard under Xp 64.

where did you snag the chipset driver for this Gigabyte mobo ?

**Comment by enigma131** [comment]
Xp is installed Under UEFI on Gpt partition.
Uefi loader is longhorn version, and no real graphic driver, I use VBE miniport driver, it acts like a GOP wrapper.
![devmgmt](https://github.com/Feodor2/Mypal68/assets/31815445/c67070eb-8260-4567-bb9c-8c663e15bac2)

**Comment by andika207** [comment]
> no real graphic driver

the chipset driver features other things aside from the VGA driver
I can't see a yellow triangle symbol so I wonder how you did manage to get it to work on XP without official drivers

what's the transfer speed on the USB 3.0 ports ? 
can you show a Crystal Disk Mark screencap ?
did you slipstream ACPI.sys file with Patch Integrator utility ?

**Comment by enigma131** [comment]
Yes of course i have modified acpi.sys, without it Xp diden't start on this generation of chipsets (and Fernando's AHCI's driver)
An why should I show you more information, you give me no information about your material since the begin of this tread, seriously ?

**Comment by andika207** [comment]
> An why should I show you more information, you give me no information about your material since the begin of this tread, seriously ?

because your request was **totally useless** for this issue and the fix I found on google has proven me right.

I want to see if your USB 3.0 ports are really working at USB 3.0 transfers speeds because the fact of the driver being displayed correctly on the device manager means nothing

**Comment by enigma131** [comment]
Score are little under W10 on same machine, I think it's comming from Exfat driver, not optimized under Xp:

![CrystalSandisk](https://github.com/Feodor2/Mypal68/assets/31815445/8c7f8ee7-13c7-4e20-8175-a06fc6887748)

**Comment by andika207** [comment]
> Scores are little under W10 on same machine

I don't use W10 so it's irrelevant here how much it scores on it
is this a 64GB pendrive ? 
I have managed to install every USB 3 driver available in the Patch Integrator folder into different OEM computers but couldn't get past USB 2.0 transfer speeds. :(

I believe some OEM computers are pure garbage allergic to the modding and other software utilities. I have never attempted to install XP on a more modern computer but will try to do it on a ASUS Prime 460 mobo

**Comment by enigma131** [comment]
Yes it is a 64 Byte one. It's Ventoy formated , only a second Fat 32 little partition on it.

**Comment by andika207** [comment]
>  On the 460, you will need the hal patch for sure, without it you can't scrol. Gecko based browsers use a lot function QuerperformanceCounter api function, including for scrooling.

OK. je voulais juste expérimenter j'ai pas besoin d' utiliser du hardware moderne pour WinXP
non, je ne suis pas french et je n'a habite pas dans ton pays nonplus au cas ou tu le demandes MDR.
c'est terribile de voir des images de ton pays, voilà le résultat de laisser passer tous les arabes et noirs du tiers monde. **MACRON et URSULA DIMISSION !**

**Comment by jonm58** [comment]
> > On the 460, you will need the hal patch for sure, without it you can't scrol. Gecko based browsers use a lot function QuerperformanceCounter api function, including for scrooling.
> 
> OK. je voulais juste expérimenter j'ai pas besoin d' utiliser du hardware moderne pour WinXP non, je ne suis pas french et je n'a habite pas dans ton pays nonplus au cas ou tu le demandes MDR. c'est terribile de voir des images de ton pays, voilà le résultat de laisser passer tous les arabes et noirs du tiers monde. **MACRON et URSULA DIMISSION !**

Don't get political on Github.

**Comment by enigma131** [comment]
> Don't get political on Github.

+1
